db/migrate/20220128144148_add_converted_data_hash_to_issues.rb

```diff
@@ -0,0 +1,6 @@
+class AddConvertedDataHashToIssues < ActiveRecord::Migration[6.0]
+  def change
+    add_column :issues, :converted_data_hash, :jsonb, index: { using: :gin },
```

Comment:
Sería algo así lo que te planteo arriba:
```suggestion
    add_column :issues, :canonical_data, :jsonb, index: { using: :gin, opclass: :gist_trgm_ops },
```
Lo saqué de https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_index

Reply:
genial